### PR TITLE
Show Requester and Maker names in Admin Requests list

### PR DESCRIPTION
**Pull request by ConnorDY**
image

image

### CODE REVIEW

File: ui/src/views/RequestFormView.tsx [hunk 1]
```diff
@@ -174,8 +174,11 @@ const RequestFormView: React.FC<{ user: User; role: string }> = ({
                     </Dropdown.Toggle>
 
                     <Dropdown.Menu>
-                      {statuses.map((status) => (
-                        <Dropdown.Item onClick={() => setStatus(status)}>
+                      {statuses.map((status, index) => (
+                        <Dropdown.Item
+                          key={index}
```
Comment (mjoyce91) [inline ui/src/views/RequestFormView.tsx:179]: I'm only just now noticing that we're using [index as key](https://reactjs.org/docs/lists-and-keys.html#keys) everywhere

Comment (ConnorDY) [reply ui/src/views/RequestFormView.tsx:179]: It's perfectly fine for short lists of non-database items, but I agree that we should replace this with the Mongo ObjectID where possible.

File: ui/src/views/MakerVerificationView.tsx [hunk 2]
```diff
@@ -216,7 +216,7 @@ const MakerVerificationView: React.FC<{ user: User }> = ({ user }) => {
                     >
                       <option value={''}>Choose...</option>
                       {states.map((state, i) => (
-                        <option key={i}>{state}</option>
+                        <option key={`state-${i}`}>{state}</option>
```
Comment (mjoyce91) [inline ui/src/views/MakerVerificationView.tsx:219]: This doesn't really change the problem with using index

Comment (ConnorDY) [reply ui/src/views/MakerVerificationView.tsx:219]: I'm aware but this is a short list of items not pulled from the database and we shouldn't be re-rendering this.

Comment (ConnorDY) [reply ui/src/views/MakerVerificationView.tsx:219]: Fixing it regardless for these items.

File: api/controllers/makers.ts [hunk 3]
```diff
@@ -72,7 +72,8 @@ export default class MakersController {
 
           <p style="font-style: italic;">
             <span style="font-weight: bold">Note:</span>
-            When you return to the Collective Shield website, you may need to log out and log back in to activate your account.
+            When you return to the <a href="https://collectiveshield.org">Collective Shield</a> website,
+            you may need to log out and log back in to activate your account.
```
Comment (mjoyce91) [inline api/controllers/makers.ts:76]: Do we need to be more explicit by linking to connect.collectiveshield.org?

Comment (ConnorDY) [reply api/controllers/makers.ts]: Discussed this with Amy and we decided it should link to the app and not the Squarespace site.

**Pull request merged**